### PR TITLE
Support for GLM-4-9B-Chat-hf and GLM-4v-9B models on vLLM >= 0.6.3 and transformers >= 4.46.0

### PR DESCRIPTION
**Pull request by sixsixcoder**
Support for GLM-4-9B-Chat-hf and GLM-4v-9B models on vLLM >= 0.6.3 and transformers >= 4.46.0, the major changes are in`GLM-4/basic_demo`

### CODE REVIEW

**Review by zRzRzRzRzRzRzR — APPROVED** [review]

File: basic_demo/glm_server.py
Comment (zRzRzRzRzRzRzR) [inline basic_demo/glm_server.py:1]: Is this args have been Removein vllm new version?

File: basic_demo/trans_cli_vision_demo.py
Comment (zRzRzRzRzRzRzR) [inline basic_demo/trans_cli_vision_demo.py:1]: Should the Int4 part of this section be retained? Can it be re-created following the example of trans_cli_demo.py?

File: basic_demo/trans_web_demo.py
Comment (zRzRzRzRzRzRzR) [inline basic_demo/trans_web_demo.py:1]: This change is correct.

File: basic_demo/glm_server.py
Comment (sixsixcoder) [reply basic_demo/glm_server.py:1]: Yes, I tested it on vllm=0.6.3.post2, and these parameters have been removed

File: basic_demo/trans_cli_vision_demo.py
Comment (sixsixcoder) [reply basic_demo/trans_cli_vision_demo.py:1]: You can merge this file later.

**Pull request merged**